test/query.test.js

```diff
@@ -1568,14 +1568,6 @@ describe('query', () => {
     return Promise.all([p1, p2, p3, p4])
   })
 
-  test('contains', () => {
```

Comment:
This test has also been removed in #330 , however, I needed to take it out here as well in order to get the tests to pass 👍